### ISSUE TITLE
Extraneous import arguments throw errors on perl 5.39.1

### ISSUE THREAD

**Issue by haarg** [wchristian/Acme-MITHALDU-XSGrabBag#2]
The code in this module is passing arguments to Acme::MITHALDU::XSGrabBag::Inline->import which includes the C code to compile. This works fine during development. For the CPAN release, there is no import method to pass the arguments to. In development versions of perl, this throws errors.

This mainly seems to be a problem in Inline::Module. I've filed a fix for that at ingydotnet/inline-module-pm#36.

**Comment by wchristian** [comment]
@haarg so right now i only need to watch that one and see when it gets merged, right?

**Comment by haarg** [comment]
Yes, that should be fine. You could manually patch the files after Inline::Module does its thing, but since this is an Acme module I doubt it's a priority.

**Comment by wchristian** [comment]
Correct, but i'm glad for the help nonetheless. :)
